tests/sshserver.pl

```diff
@@ -393,9 +393,9 @@
 
 if ($^O eq 'MSWin32' || $^O eq 'cygwin' || $^O eq 'msys') {
     # convert MinGW drive paths to Cygwin drive paths
-    $clipubkeyf_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
-    $hstprvkeyf_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
-    $pidfile_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
+    chomp($clipubkeyf_config = `cygpath -u -a $clipubkeyf_config`);
+    chomp($hstprvkeyf_config = `cygpath -u -a $hstprvkeyf_config`);
+    chomp($pidfile_config = `cygpath -u -a $pidfile_config`);
```

Comment:
These lines require a more flexible patch that does not require the cygpath utility.